src/main/kotlin/com/workos/webhooks/models/OrganizationMembershipCreatedEvent.kt

```diff
@@ -0,0 +1,20 @@
+package com.workos.webhooks.models
+
+import com.workos.usermanagement.models.OrganizationMembership
+
+/**
+ * Webhook Event for `organization_membership.created`.
+ */
+class OrganizationMembershipCreatedEvent(
```

Comment:
Would it be good to avoid having one class per event, in cases like this one, where all the classes are identical? We could have a `OrganizationMembershipEvent` class instead. Thoughts?

Reply:
Yeah great point. I'll combine!